### DISCUSSION TITLE
12 Step Meeting List in Google Sites?

### DISCUSSION THREAD

**Discussion by OperationMindcrime** [code4recovery/12-step-meeting-list#1082] [Q&A] [answered]
Hello:

I'm trying to put together a basic website for our local area.  The only thing I don't know how to do is add the meeting list to the site.  Can this be done with your plugin? If so, any tips on how to do it?  

Thanks for any help!

**Comment by kiyote33** [comment]
Hi @OperationMindcrime , we have a short video on our [website ](https://code4recovery.org/how-to/how-to-use-the-12-step-meeting-list) which might answer your question. There's also a documentation section with FAQs that you may find useful. 
Let us know if you have any more questions. We're happy to be of assistance...

**Comment by joshreisner** [comment, marked as answer]
The plugin needs WordPress, and so it can't be installed with Google Sites. However a member recently created this resource https://stack.pugetsoundaa.org/ which describes how they used Google Sites in their local area.

**Reply by OperationMindcrime** [reply]
Thank you.  I didn't realize I needed Wordpress.  I'll check out that link!
